src/datapump/sync/sync.py

```diff
@@ -723,6 +723,125 @@ def get_latest_release(self) -> Tuple[str, List[str]]:
         return latest_release, source_uri
 
 
+class DISTAlertsSync(Sync):
+    """
+    Defines jobs to create new DIST alerts assets once a new release is available.
+    """
+
+    dataset_name = "umd__glad_dist_alerts"
+    source_bucket = "earthenginepartners-hansen"
+    source_prefix = "DIST-ALERT"
+    input_calc = """
+        np.where((A>=30) & (A<255) & (B>0) & (C>=2) & (C<255),
+            np.where(C<4, 20000 + B, 30000 + B),
+            -1
+        )
+    """
+
+    def __init__(self, sync_version: str):
+        super().__init__(sync_version)
+
+    def get_latest_release(self) -> Tuple[str, List[str]]:
+        """
+        Get the version of the latest release in GCS
+        """
+
+        # Raw tiles are just updated in-place
+        source_uris = [
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-ANOM-MAX",
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-DIST-DATE",
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-DIST-COUNT",
+        ]
+
+        # This file is updated once tiles are updated
+        upload_date_text = get_gs_file_as_text(
+            self.source_bucket, f"{self.source_prefix}/uploadDate.txt"
+        )
+
+        # Example string: "Updated Fri Apr 15 14:27:01 2022 UTC"
```

Comment:
When I check uploadDate.txt, the format is:

Updated Sat Oct 26 12:22:54 2024-10-26 UTC

Not sure if it changed, but I think you need to extend the format string at line 764 to account for the '-10-26' part.  Can you test by hand in python and make sure this parsing will work?